src/codemagic/tools/app_store_connect.py

```diff
@@ -148,6 +161,81 @@ def _delete_resource(self, resource_manager, resource_id: ResourceId, ignore_not
             else:
                 raise AppStoreConnectError(str(api_error))
 
+    @cli.action('list-builds',
+                BuildArgument.APPLICATION_ID_RESOURCE_ID_OPTIONAL,
+                BuildArgument.EXPIRED,
+                BuildArgument.NOT_EXPIRED,
+                BuildArgument.BUILD_ID_RESOURCE_ID,
+                BuildArgument.PRE_RELEASE_VERSION,
+                BuildArgument.PROCESSING_STATE,
+                BuildArgument.BUILD_VERSION_NUMBER)
+    def list_builds(self,
+                    application_id: Optional[ResourceId] = None,
+                    expired: Optional[bool] = None,
+                    not_expired: Optional[bool] = None,
+                    build_id: Optional[ResourceId] = None,
+                    pre_release_version: Optional[str] = None,
+                    processing_state: Optional[BuildProcessingState] = None,
+                    build_version_number: Optional[str] = None,
```

Comment:
I still find that this should come in as integer.
```suggestion
                    build_version_number: Optional[int] = None,
```